random_menu.rb

```diff
@@ -0,0 +1,54 @@
+# Declare menu items
+array_adjective = [ "Hot",
+  "Cold",
+  "Spicy",
+  "Soft",
+  "Hard",
+  "Salty",
+  "Sweet",
+  "Creamy",
+  "Crunchy",
+  "Milky"
+]
+
+array_style = [ "Steamed",
+  "Pan-fried",
+  "Sauteed",
+  "Sezchuan",
+  "Curried",
+  "Teriyaki",
+  "Tandoori",
+  "Chopped",
+  "Seasoned",
+  "Roasted"
+]
+
+array_food = [ "Tofu",
+  "Seitan",
+  "Potatoes",
+  "Bread",
+  "Beans",
+  "Rice",
+  "Eggplant",
+  "Mushroom",
+  "Noodles",
+  "Peppers"
+]
+
+# Request and receive user input for number of items
+print "Welcome to Sahana's Crazy Menu!\nHow many items would you like to peruse? We have 10 items. "
+user_input = gets.to_i
+
+# Verify user input
+# Generate random Menu
+# Delete items from arrays so they cannot be used again
+if user_input <= 10
+  user_input.times { |n|
+    puts "#{ n + 1 }. #{adjective = array_adjective.sample} #{style = array_style.sample} #{food = array_food.sample}"
```

Comment:
I like the idea of storing the chosen menu components in variables and then removing them from the lists. However, assigning a variable inside of a string interpolation is a little hard to read. Could clean this up by breaking it into multiple lines?